### PR TITLE
Body scanner bugfixes and alt clicking

### PR DESCRIPTION
**Pull request by nauticall**
Fixes two bugs with body scanner print outs:
1. Body scanners displaying body temperature wrong (and showing something like -240 degrees C). Doesn't show Kelvin anymore because the Kelvin readout is useless and can confuse people.
2. Body scanners not showing shrapnel (or _any_ implant for that matter) on the printouts. Turns out the code saves all the implants/shrapnel to a list and then just... never printed out that list. Weird.

In addition, also tweaks some wording so that the paper printouts from the body scanners will mimic the GUI scans more accurately.

Also added alt-click eject functionality to the body scanner and sleeper, because we're on this path so we might as well double down.

### CODE REVIEW

**Comment by nauticall** [comment]
!review

File: code/game/machinery/body_scanner.dm [hunk 1]
```diff
@@ -525,7 +529,7 @@
 				else
 					unk += 1
 			if (unk)
-				wounds += "has an abnormal mass present"
+				wounds += "has unknown object(s) present"
```
Comment (SleepyGemmy) [inline code/game/machinery/body_scanner.dm:532]:
```suggestion
				wounds += "has unknown objects present"
```
plural as default here is fine.

File: code/game/machinery/body_scanner.dm [hunk 2]
```diff
@@ -727,22 +731,20 @@
...
 
 		if (e.implants.len)
-			var/unknown_body = 0
+			var/unk = 0
 			var/list/organic = list()
-			var/imp
-			for(var/I in e.implants)
-				if(is_type_in_list(I,known_implants))
-					imp += "[I] implanted:"
+			for (var/atom/movable/I in e.implants)
```
Comment (SleepyGemmy) [inline code/game/machinery/body_scanner.dm:736]:
```suggestion
			for(var/atom/movable/I in e.implants)
```

File: code/game/machinery/body_scanner.dm [hunk 3]
```diff
@@ -727,22 +731,20 @@
...
+				if(is_type_in_list(I, known_implants))
+					wounds += "\a [I.name] implanted:"
 				else if(istype(I, /obj/effect/spider))
 					organic += I
 				else
-					unknown_body++
-			if(unknown_body)
-				imp += "Unknown body present:"
+					unk += 1
+			if (unk)
```
Comment (SleepyGemmy) [inline code/game/machinery/body_scanner.dm:743]:
```suggestion
			if(unk)
```

File: code/game/machinery/body_scanner.dm [hunk 4]
```diff
@@ -727,22 +731,20 @@
...
+				wounds += "has unknown object(s) present:"
```
Comment (SleepyGemmy) [inline code/game/machinery/body_scanner.dm:744]:
```suggestion
				wounds += "has unknown objects present:"
```
plural as default here is fine.

**Comment by nauticall** [comment]
!bugfix

**Review by alsoandanswer — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

File: code/game/machinery/Sleeper.dm [hunk 5]
```diff
@@ -365,6 +365,10 @@
 	toggle_filter()
 	toggle_pump()
 
+/obj/machinery/sleeper/AltClick()
+	if(usr.Adjacent(src))
```
Comment (NonQueueingMatt) [inline code/game/machinery/Sleeper.dm:369]: use_check_and_message

File: code/game/machinery/body_scanner.dm [hunk 6]
```diff
@@ -88,6 +88,10 @@
 	add_fingerprint(usr)
 	return
 
+/obj/machinery/bodyscanner/AltClick()
+	if(usr.Adjacent(src))
```
Comment (NonQueueingMatt) [inline code/game/machinery/body_scanner.dm:92]: use_check_and_message

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/machinery/Sleeper.dm [hunk 7]
```diff
@@ -365,6 +365,10 @@
 	toggle_filter()
 	toggle_pump()
 
+/obj/machinery/sleeper/AltClick()
+	if(usr.use_check_and_message(src))
```
Comment (SleepyGemmy) [inline code/game/machinery/Sleeper.dm:369]:
```suggestion
	if(use_check_and_message(usr))
```

File: code/game/machinery/body_scanner.dm [hunk 8]
```diff
@@ -88,6 +88,10 @@
 	add_fingerprint(usr)
 	return
 
+/obj/machinery/bodyscanner/AltClick()
+	if(usr.use_check_and_message(src))
```
Comment (SleepyGemmy) [inline code/game/machinery/body_scanner.dm:92]:
```suggestion
	if(use_check_and_message(usr))
```

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**